src/slo/x_days_sli/sli_view_querier.py

```diff
@@ -0,0 +1,54 @@
+import logging
+
+from apiclient.errors import HttpError
+
+from src.commons.big_query.big_query import BigQuery
+from src.commons.table_reference import TableReference
+
+
+class SLIViewQuerier(object):
+    def __init__(self):
+        self.bigQuery = BigQuery()
+
+    def query(self, x_days):
+        query_results = self.bigQuery.execute_query("""SELECT * FROM [project-bbq:SLO_views_legacy.SLI_3_days]""")
```

Comment:
Please retrieve this id from configuration

Reply:
done